src-electron/sdk/matter.js

```diff
@@ -163,19 +150,20 @@ async function computeStorageImport(
   forcedExternal,
   attributeName
 ) {
-  if (
-    forcedExternal.byName &&
-    forcedExternal.byName[clusterName] &&
-    forcedExternal.byName[clusterName].includes(attributeName)
-  ) {
-    storagePolicy = dbEnum.storagePolicy.attributeAccessInterface
-  }
+  forcedExternal.forEach((option) => {
+    if (
+      option.optionCategory == clusterName &&
+      option.optionLabel == attributeName
+    ) {
+      storagePolicy = dbEnum.storagePolicy.attributeAccessInterface
```

Comment:
Need not go through the rest of the elements in the `forcedExternal` once the match has been found. You can break out of the forEach loop using the `return` statement here.
Similar change needs to happen in the above .map function calls above

Reply:
good point, about to push a fix